src/Components/RecsListTable/index.js

```diff
@@ -296,6 +309,18 @@ const RecsListTable = () => {
     },
   ];
 
+  const onSort = (_e, index, direction) => {
+    const sortRecsDirection =
+      direction === SortByDirection.asc
+        ? buildFilteredRows(recs, filters, index)
```

Comment:
Again, what about calling just buildDisplayedRows and providing the sorting parameters to this function? So that the function 1) sorts the rows that were already pre-filtered by buildFilteredRows, and 2) makes a slice of this array to meet pagination params (already implemented).